README.markdown

```diff
@@ -243,6 +243,10 @@ You can provide multiple hostnames, separated by commas.
 lr.docker.environment.web.server.hostnames=localhost
 ```
 
+#### Use custom JVM Arguments
+
+The default JVM arguments for running the Liferay server can be overriden inside of `./liferay-jvm-opts.env`. You can also add other JVM arguments as long as it is appended to LIFERAY_JVM_OPTS. There are pre-existing overrides already in this file to help improve with Liferay server performance.
```

Comment:
Two things here:
1. This is in the Webserver section, it should probably instead be in either its own section or in a section related to the Liferay features
2. It does not have a corresponding entry in the table of contents above. Users should be able to navigate using the TOC links